### PR TITLE
Upgrade SDK and project

### PR DESCRIPTION
**Pull request by andrei-gorin**
- SDK version 0.18.0
- The project was replaced with new one created with XCode 11.2.1
- Readme updated with better instructions and known issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,42 +7,57 @@
...
+	* `$(PROJECT_DIR)/Carthage/Checkouts/ios-webrtc/output/bitcode`
+	* `$(PROJECT_DIR)/Carthage/Checkouts/ios-bundle`
 
+##### To link the files with older XCode:
 * In XCode, click your project in the *Project Navigator*. Select your *target*, choose the *General* tab at the top, and scroll down to the *Linked Frameworks and Libraries* section at the bottom.
-* Drag and drop all your `*.framework` files (except `RxTest.framework`) from *Carthage/checkouts* and *Carthage/build* folders to that section in XCode.
+* Drag and drop all your `*.framework` files (except `RxTest.framework` and any other test related frameworks) from *Carthage/checkouts* and *Carthage/build* folders to that section in XCode.
 
-#### 4) Add a Run script for Carthage
+##### Add a Run script for Carthage (Not required with XCode 11):
```
Comment (gugalo) [inline README.md]: I would move the part from the end of the header to the beginning maybe even rephrase:
(For XCode 10 and below) Add a Run script for Carthage

**Comment by andrei-gorin** [comment]
> I see some changes which are probably because of re-creating the project with newer XCode, but why there are new UI elements:
> `ContentView` and `SceneDelegate` or they are created by default?

Yes, they are created by default when you create new SwiftUI enabled project.

**Comment by ostap0207** [comment]
Are they needed? Can we delete them?

File: README.md (hunk 1 above)
Comment (andrei-gorin) [reply README.md]: Updated

**Comment by andrei-gorin** [comment]
> Are they needed? Can we delete them?
We should not delete em.

**Comment by ostap0207** [comment]
> We should not delete em.

Could you elaborate why not and why they are needed?

**Comment by andrei-gorin** [comment]
> > We should not delete em.
> 
> Could you elaborate why not and why they are needed?

https://salemove.slack.com/archives/GPLEK7UGY/p1579167961000100?thread_ts=1578919252.006200&cid=GPLEK7UGY

**Pull request merged**